docs/sources/tempo/configuration/grafana-alloy/tail-sampling/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Sampling
+menuTItle: Sampling
+description: Use sampling to optimize sampling decisions
+weight:
+aliases:
+- /docs/tempo/grafana-alloy/tail-based-sampling
+- ../tail-based-sampling/
+---
+
+# Sampling
```

Comment:
i'm wondering about the positioning of the sampling concepts - since sampling isn't just tempo, right? it's a technique used across the different signals, too? maybe we say Grafana rather than Grafana Tempo, too?